src/select/SelectDropdown.ts

```diff
@@ -28,16 +34,23 @@ const useSelectDropdown = createHook<SelectDropdownOptions, BoxHTMLProps>({
   useProps(
     {
       maxHeight = 500,
-      move,
       items,
       selected,
-      setSelected,
-      setCurrentId,
       typehead,
       visible,
+      isCombobox,
+      move,
+      setSelected,
+      setCurrentId,
+      setTypehead,
     },
     { ref: htmlRef, style: htmlStyle, ...htmlProps },
   ) {
+    const { handleOnKeyPress } = useTypeahead({
+      setTypehead,
+      disabled: isCombobox,
```

Comment:
Disabling typeahead in combobox while the dropdown is open, so that it does not interfare with combobox's input value.